src/com/xilinx/rapidwright/edif/EDIFTools.java

```diff
@@ -382,14 +382,29 @@ public static String makeNameEDIFCompatible(String currName) {
     }
 
     /**
-     * Strips off bracket index in a bussed name (ex: {@code "data[0]" --> "data"}).
+     * Strips off bracket index in a bussed name (ex:
+     * {@code "data[0]" --> "data["}). Leaves open bracket by default as this is the
+     * key for bussed ports in the port map of {@link EDIFCell}
+     * 
      * @param name Bracketed bussed name.
-     * @return Name of bus with brackets removed
+     * @return Name of bus with index and close bracket removed
      */
     public static String getRootBusName(String name) {
```

Comment:
This method looks similar to `getStemName()`. Perhaps we can deprecate this (it doesn't appear to be used in RapidWright, but is used once elsewhere) while we're here?

Reply:
Which method do you mean to deprecate (or both)?

Reply:
Sorry, it looks like I'm confusing `EDIFPort.getStemName()` and `EDIFPort.getBusName()` (what is the difference between these two?) with `EDIFTools.getRootBusName()`.

I'm okay with it if you feel like this out of scope of this PR, but the synonym-y nature of all three is a bit confusing.

Reply:
Deprecating `EDIFPort.getStemName()`